app/views/podcasts/_form.html.erb

```diff
@@ -48,11 +50,11 @@
       </div>
       <div class="field">
         <%= form.label :image %>
-        <%= form.file_field :image %>
+        <%= form.file_field :image, accept: "image/*" %>
```

Comment:
Adding the `accept` argument really enhances the UX. It limits file selection to the specific file type.